### PR TITLE
Do not automatically open debug tools

### PR DESCRIPTION
**Pull request by octacian**
This PR replaces the dependency to `electron-debug` which caused the developer debug tools to be automatically opened at window creation with a menu entry under the `View` section (triggered with `Ctrl+Shift+I` on Windows/Linux or `Alt+Shift+I` on MacOS).

This is an improvement to the previous functionality as I have personally found myself having to close the developer debug tools almost every single time I launch a developer version of the application. Not only that, but considering the higher limitations to resizing placed on Ramme's windows, it can at times be impossible to close the debug tool with too small a window.

TL;DR: Automatically opening developer debug tools was annoying; this PR removes the feature and replaces it with the ability to easily open debug tools when *you* want to.

### CODE REVIEW

**Comment by genbtc** [comment]
Why did this fail the appveyor check ?

**Comment by octacian** [comment]
> Why did this fail the appveyor check ?

Quite honestly, I'm not sure. I can't seem to locate anything that *I* did that would cause the build to fail. Still getting used to AppVeyor, so, I really don't know.

**Comment by octacian** [comment]
Nope, I cannot for the life of me figure out why AppVeyor is failing. Even read the entire thing through line-by-line, and didn't see anything regarding this PR. If someone could enlighten me, I'd really appreciate it.

**Comment by bil-elmoussaoui** [comment]
It must be just an issue with not being able to get a sub-dependency. Can you just push another commit to force appveyor to relaunch?

**Comment by octacian** [comment]
@bil-elmoussaoui AppVeyor build seems to still be failing. I notice, however, that the main project AppVeyor build is also failing, so this must be a dependency issue outside of this PR.

**Comment by terkelg** [comment]
Can you have a look at the merge conflict. Also, you should join the Ramme Dev slack.

**Comment by octacian** [comment]
> Can you have a look at the merge conflict. Also, you should join the Ramme Dev slack.

Will do. How can I join the Ramme Slack? My understanding is that I need an email invite in order to join if that's the case, my email can be found on my GitHub profile page.

**Comment by octacian** [comment]
Fixed merge conflict..

**Comment by terkelg** [comment]
Thanks guys. Can you bump version do a new release @bil-elmoussaoui so Ramme can auto-update.

**Comment by bil-elmoussaoui** [comment]
@terkelg We still need to fix Appveyor so we can get working windows build. And I don't have access to AppVeyor. Maybe @anatolinicolae can help on that?
